src/components/themed-image/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { ThemedImageProps } from './types'
+
+/**
+ * This component can be used in views where we need to
+ * conditionally render an image based on the theme state.
+ *
+ * See styles/themes/global.css for the attribute handler
+ * that uses `display:none` to conditionally render content
+ * based on the theme state with css
+ */
+export default function ThemedImage({ src, alt }: ThemedImageProps) {
+	/**
+	 * @TODO could add protection here to throw
+	 * when used outside of the theme context. Since the
+	 * entire app is wrapped with the provier when using the
+	 * core devdot layout, it's currently low-risk
+	 * */
+	return (
+		<>
+			{/* When the theme is dark, hide this div */}
+			<div data-hide-on-theme="dark">
```

Comment:
Should use `GlobalThemeOption` here

Reply:
Also this only allows two themes....should we do a 'show-on-theme' instead?

Reply:
Maybe I don't want the wrapper div 💭 

also this could use Image and accept next/image props